src/WebJobs.Script.Scaling/ScaleUtils.cs

```diff
@@ -89,14 +88,20 @@ public static string ToDisplayString(this IEnumerable<string> values)
         public static string GetToken(DateTime expiredUtc)
         {
             var bytes = BitConverter.GetBytes(expiredUtc.Ticks);
-            var encrypted = MachineKey.Protect(bytes, Purpose);
+
+            // TODO: FACAVAL: Get machine key (use same approach as data protection)
```

Comment:
You're going to address this before checking in?

Reply:
Yes.